meshmode/discretization/visualization.py

```diff
@@ -339,7 +339,7 @@ class Visualizer(object):
     .. automethod:: show_scalar_in_mayavi
     .. automethod:: show_scalar_in_matplotlib_3d
     .. automethod:: write_vtk_file
-    .. automethod:: write_high_order_vtk_file
+    .. automethod:: write_par_vtk_file
```

Comment:
```suggestion
    .. automethod:: write_parallel_vtk_file
```
Since it's not that long in full?

Reply:
74f74daa50d2051b521c01fa529708501ea28da2